session/session.go

```diff
@@ -2777,6 +2786,7 @@ func createSessionWithOpt(store kv.Storage, opt *Opt) (*session, error) {
 	// session implements variable.GlobalVarAccessor. Bind it to ctx.
 	s.sessionVars.GlobalVarsAccessor = s
 	s.sessionVars.BinlogClient = binloginfo.GetPumpsClient()
+	s.sessionVars.StmtStats = s.stmtStats
```

Comment:
I'm worried that the StmtStats in sessionVars is only assigned here but not in places where SessionVars are constructed. It may cause problems if other codes are controlling the session vars.